Utils/Dataflow/025_chicagoES/stage.py

```diff
@@ -51,7 +54,7 @@ def init_es_client():
     chicago_es = elasticsearch.Elasticsearch(chicago_hosts)
 
 
-def task_metadata(taskid, fields=[]):
+def task_metadata(taskid, fields=[], retry=3):
```

Comment:
I added it after stumbling on `TimeoutException` :) It is not  that critical to stop processing, I thought -- we can wait a little and try again.

In the ideal world, where stages are run by Kafka supervisor, there\`s no need in retry: in case of failure supervisor itself would retry message processing few times before skipping it.
But now it is run by shell script, that starts every hour or so -- and in case of a failure it will restart only an hour from "now". Not good by itself; and if there\`re some network issues -- then we risk to never reach most resent records in our processing.
So I thought, adding this functionality :)

Reply:
@mgolosova 

Everything is clear now, thank you!